css/ball/main.scss

```
@@ -121,6 +121,14 @@ body {
         }
     }
 
+    .dropdown {
+      font-family: $font-secondary;
+    }
+
+    .dropdown-content {
+      background-color: $secondary-lowlight;
+    }
```

```suggestion
    .dropdown-content {
      background-color: $secondary-lowlight;
      
      a {
        font-family: $font-secondary;
      }
    }
```

Just to make the font of the dropdown menu the same as the rest of the ball pages